### ISSUE TITLE
Rule Change: (fill in)

### ISSUE THREAD

**Issue by antoniotrkdz** [eslint/eslint#15369]
### What rule do you want to change?

key-spacing "align": { "beforeColon": true, "afterColon": true, "on": "colon" }

### What change to do you want to make?

Implement suggestions

### How do you think the change should be implemented?

A new option

### Example code

```js
// Actual result
book: {
    title       : 'A Book'
    , pDate       : '2021'
    , toc         : 'TABLE OF CONTENTS'
    , description : 'Description'
  }

// should be
book: {
    title         : 'A Book'
    , pDate       : '2021'
    , toc         : 'TABLE OF CONTENTS'
    , description : 'Description'
  }

// Or
book: {
      title       : 'A Book'
    , pDate       : '2021'
    , toc         : 'TABLE OF CONTENTS'
    , description : 'Description'
  }
```

### What does the rule currently do for this code?

The rule does not align correctly the first item when 'comma stile first' rule is also active.
The first item colon is positioned 2 spaces behind the following items because of not having the comma and the space.
If the first colon is manually moved to the desired position the indent rule gets triggered complaining about two more spaces. 

### What will the rule do after it's changed?

The rule should indent the first element colon two spaces to the right in order to align to the following (comma first) elements colon.

### Participation

- [ ] I am willing to submit a pull request to implement this change.

### Additional comments

There was an issue about this long time ago: #3877, but the solution(s) proposed there are not viable or not working anymore.

**Comment by nzakas** [comment]
Thanks for the input. As [stated in the README](https://github.com/eslint/eslint#stylistic-rule-updates), we actually aren't making any further updates to rules that enforce stylistic preferences. If you have a strong preference for such a change, you can always copy the existing rule into a custom rule and make it behave the way you want.

**Comment by antoniotrkdz** [comment]
OK, I will try to do that. Thank you.
